### PR TITLE
feat: Update TubePlayer to SingleProject

### PR DESCRIPTION
**Pull request by eriklimakc**

### CODE REVIEW

File: reference/TubePlayer/src/TubePlayer/TubePlayer.csproj [hunk 1]
```diff
@@ -0,0 +1,50 @@
+<Project Sdk="Uno.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>
+      $(DotNetVersion)-browserwasm;
+      $(DotNetVersion)-android;
```
Comment (jeromelaban) [inline reference/TubePlayer/src/TubePlayer/TubePlayer.csproj:5]: Can you add iOS?

**Review by jeromelaban — APPROVED** [review]

File: reference/TubePlayer/src/TubePlayer/TubePlayer.csproj [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
+    </TargetFrameworks>
+
+    <OutputType>Exe</OutputType>
+    <SingleProject>true</SingleProject>
```
Comment (eriklimakc) [inline reference/TubePlayer/src/TubePlayer/TubePlayer.csproj:14]:
```suggestion
    <UnoSingleProject>true</UnoSingleProject>
```

File: reference/TubePlayer/src/TubePlayer/TubePlayer.csproj [hunk 3]
```diff
@@ -0,0 +1,52 @@
+<Project Sdk="Uno.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-browserwasm;
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0;
```
Comment (eriklimakc) [inline reference/TubePlayer/src/TubePlayer/TubePlayer.csproj:10]:
```suggestion
      net8.0-desktop;
      net8.0-android;
      net8.0-ios;
      net8.0-maccatalyst;
      net8.0-windows10.0.19041;
      net8.0-browserwasm;
```

**Comment by agneszitte** [comment]
Closed as the work will be done in https://github.com/unoplatform/Uno.Samples/pull/653

**Pull request closed without merging**